### ISSUE TITLE
More ATOM integration

### ISSUE THREAD

**Issue by jaekwon** [atomone-hub/genesis#12]
We could allow some ATOM to bid for ATOM1 up to a limit. If we 100% slash YES voters for ATOM1 then letting ATOM bid for half of ATOM1 seems pretty reasonable too.

**Comment by AloisLIEN-SetC** [comment]
Another view is not to slash YES voters at all, but to award a bonus to NO voters and NWV voters so that the latter - all of us - have more weight in governance from the start.

It's a question of perspective. We do not begin the story of ATOM1 with a penalty for people who think via the brains of influencers but with a bonus for those who value their sovereignty. We remain with a view of positivity.

**Comment by giunatale** [comment]
I think finding true alignment is not just a matter of looking at voters of proposal 848. We had other governance proposals that could be used to refine the filtering, prop 69 and prop 82 for example. Future ATOM1 distribution should be weighted on how you voted on all those IMHO. But yes, slashing initially should be done or the risk you mentioned can possibly materialize. I am possibly even advocating for more aggressive slashing here that slashes even prop 848 no-voters that e.g. voted yes on prop 82.
My guess is it could allow to identify with more confidence people that would support the vision.

I am also personally mostly in favor of some post-genesis acquisition method. I am a bit afraid though of the prospect of attracting the same crypto "degens" that made 848 pass on the Hub in the first place. But I guess this is a risk that will always be there. 
Maybe some incentive to bring ATOM over to get phATOM? this also in order to bootstrap governance power for the partyhub. You might get some phATOM1 too so without governance voting power? I don't know maybe I am talking silly.

**Comment by jaekwon** [comment]
Thank you for your feedback. This feedback is good, we can do surveys but we do need validation for this to work. I agree with 82. Maybe we make it 848 and 82 as pure proposals, 69 was "biased". 

phATOM* anything phat.

**Comment by jaekwon** [comment]
Besides framing all of this in terms of rewards rather than slashing, which I think is great overall because (a) it legitimately is arguably an incentive because the vote is split and (b) it prevents the worst discord between the splits because one side is completely removed; there is still the question of how much reward an ABSTAIN/NO-voter should get, and it seems that the answer should depend on the actual ratio of YES:NO+NWV and no-voters can inherit the blend of rewards (and this is "complete" in the sense of accountability by some measure); and if we wanted to we could maybe add a *slight* did-not-vote penalty on top.

My point is that ABSTAIN/no-vote does have a good base model for how to calculate genesis distribution.

**Comment by AloisLIEN-SetC** [comment]
And imo, to be totally fair, we should distinguish those who forgot to vote the prop 848 from those who never vote because they don't care (looking for example at props 69 and 82).

**Comment by jaekwon** [comment]
Aren't they punished enough from inflation? Since they get punished from inflation aren't we able to give them a bit of protection as a class? Some people need to unstake momentarily just to redelegate a third time, or for other reasons, like to move to another address.

**Comment by diegocreta** [comment]
For me the best idea is to consider the participation ratio, i.e. if the chain is protected with 2/3 of the tokens in staking, we can airdrop 2/3 +5% to those who voted NO to Prop 848 and 1/3 -5% to those who voted yes. This way if ATOM1 reaches the safe threshold it will be that at least ALL people who voted "no" agree with airdrop and it will be a huge signal of strength. The best and great result for the new ATOM1! On the contrary, it is the risks of work, all or nothing.

**Comment by diegocreta** [comment]
Another Idea is to constrain who vetod YES to prop 848 to claim directly phATOM1, the only interest in persone against ATOM1 is to sell the fork, in this way will sell only the LSK token without damage the chain !

**Comment by giunatale** [comment]
> For me the best idea is to consider the participation ratio, i.e. if the chain is protected with 2/3 of the tokens in staking, we can airdrop 2/3 +5% to those who voted NO to Prop 848 and 1/3 -5% to those who voted yes. 

@diegocreta In my mind this is not as simple. distributing ATOM1 like that doesn't guarantee in some sense that governance (which is not strictly related to block production which is where the 2/3 target staking ratio matters) will be aligned. But maybe completely slashing to 0 no voters might be too harsh? yet I agree with the initial post from jae where he states that from first principles, slashing them to 0 is the safest and most logical approach.

> Another Idea is to constrain who vetod YES to prop 848 to claim directly phATOM1, the only interest in persone against ATOM1 is to sell the fork, in this way will sell only the LSK token without damage the chain !

I am not entirely against doing something like that. In general, since phATOM1 -> ATOM1 conversion would be somewhat rate-limited (i.e. you can't convert phATOM1 to ATOM1 like a regular unbonding), at least it would limit impact on governance since phATOM1 would have no governance rights (and it would require time to convert all of them back to ATOM1, stake them and get governance rights).

> I feel there should be a priority to no/nwv. I believe there should be a slash penalty for yes as outlined. Vesting options as well. I like the ideas for various snapshots within the vote, as there was much movement to no when the whiff of an airdrop occurred. The current solution is simple enough (always apply Occam's Razor) that it will be able to be explained. The rationale needs to be tight for this to be palpable.

@Antimodez appreciate your opinion, "competent layman" is kinda funny. Yes it should have been pretty easy to undestand for anyone in this industry that is past the "beginning stages". But it's also true some people are blinded by greed and "less supply means numba go up cause less selling pressure" also was a pretty easy to understand message, and an enticing narrative for the more naive retail user. Anyway, wanted to comment something more specific about your quoted text above.
If we want to "keep it simple" doing multiple snapshots during the voting period is kinda not doing that. I understand your frustration regarding airdrop chasers, but at the same time we can't assume everyone that switched did it for that. I think just one snapshot at the end of the voting period is sufficient.
Airdrop won't be perfect, it's impossible. We just need to do our best to build a governance community that is as much aligned as possible to the vision.

**Comment by jaekwon** [comment]
At the risk of getting people upset because of uncertainty, I'll stake an initial try at this.

1. Everything is phrase as a reward. This keeps it positive (and since the YES/NO was split around the middle, it is not just sugar coating it is legitimate) and also ensures that wrong-voters get at least some modicum of stake by default. And this type of learning-encouraging-forgiveness seems good especially in these early days.
2. You inherit your delegatee/validator's vote with no penalty.
3. If you voted YES, you get no reward, but you get the same amount of $ATOM1 as $ATOM. 
4. If you voted NO, you get a 2x reward, so $ATOM1 == $ATOM x3. 
5. If you voted NWV, you get the same as NO, but 5% more, or 10% if the NWV did succeed in veto.
6. If you voted ABSTAIN, or you did not vote, or you were not staked, in all of these cases you get the blend of YES/NO of those who did vote, so if everyone else voted NO then you get the NO reward; but with 10% penalty for not voting on something important.

For #4, with 3x (2x reward) for NO voters, given that Y=73.165M voted YES and N=68.336M voted NO+NWV,  the resultant distribution would have voted NO on prop 848 with 73.70% of the voting power, but with 2x (1x reward) for NO voters the resultant distribution would have voted NO with 65.13% of the voting power. Notice that both 73.70% and 65.13% are both under the Constittuional Majority of 90%. In order to reach the Constitutional Majority of 90% we would need the reward to be 9x (very high). So do we make it that high?

Otherwise, how can we possibly hope to achieve a constitutional majority? I mean, we probably will at some time, but it would be better if we could guarantee it somehow... that eventually a constitutional majority can be achieved. Not sure how this can be done without lowering the constitutional majority which I don't recommend.

**Comment by COverS8** [comment]
Airdrop part from issue #16 brought here from @giunatale suggestion

### AIRDROP suggestion

By the mintscan explorer

List of the top 1,000 accounts with the highest holdings of ATOM is between
19.98M-26K Atom

1,634,664 accounts majority holds
0.. to 26,000 Atom

1:1 distribution wouldn't be fair in nowadays anymore because the Cosmos early contributors holds significantly more Atom than the later comers because of the Atom higher price in fiat which prevented acquiring a larger amount of Atom financially. That doesn't mean that the later smaller holders wouldn't be one of the most enthusiastic cosmonauts and would be wrong that their skin in the game would continue to be low with Atom-One hub too like in the Cosmos hub. So I suggest for sub 5,000 Atom stakeholders some kind of multilpliers that the distribution would be more equal and decentralized right from the beginning. Multipliers could consist from e.g. Atom-One contributions, Cosmos hub-4 voting history, Cosmos loyalty; staking duration since Cosmos hub-4 launch etc.

**Would be good to have seperate data out based on these 1,634,664 accounts staked lets say top 5 as follow

How many staked Atoms most of these 1,634,664 accounts holds (the avg.)?

Second greatest Atom accounts 

3rd .. 4th .. 5th**

**Comment by boliwar** [comment]
Don't you think you're overcomplicating things with these ratios and percentages and stuff?
People made a choice by voting.
Those who didn’t vote are not interested.
Who voted “YES” and take airdrop ATOM1   this  let the goat into the garden. They (YESsers) love commercial things and will buy  ATOM1.

**Comment by COverS8** [comment]
The main reason for the fork lies in the past and controversial proposals of the Cosmos hub and other controversies. While the mentioned proposals are among the most disputed, there's other proposals too to consider add to these. Disputes arose because the opposing interests solely focused just on how to make Atom monetary value to pump with whatever desperate artificially ideas or integrating experimental tech into the hub. When the hub minimalism side always consistently prioritized everything the safety first, that anything experimental or making any changes which could endanger the hub, its operations and its users shouldn't be directly integrated into the hub or be done, instead exercising these experiments outside from the hub or on own chain. Despite the disrespect on Atom and its performance in monetary value perspective which is funny from this opposite side as they suppose to be with Cosmos and Atom, Atom  thrived in 2021 and they do understand the markets 100% the fact that in 2022 the whole crypto market crashed too which resulted in the meltdown of over 2T in crypto total value. Their deep desperation and desperate unsustainable ideas stems from very long 2022-2023 bear market and nothing else. Another with proposals is community pool abuses and opaque or full lack of in transparency and accountability towards to community. So therefore bad decisions/decision makers should be slashed and good ones rewarded. Airdrop is important for adoption but should be considered well by the fundamentals of hub minimalism , but it should not be only influence for anyone to join AtomOne which is open for everyone and not depending from airdrop. Now is the right time to address all other issues that often caused disappointment in the Cosmos hub and its community. I just want to remind everyone of these fundamental reasons and the fork's idea, good if it sparks the smarter ones. But together will be decided what will be the best for AtomOne

**Comment by floridamanfintech** [comment]
I tend to agree with excluding those who have shown they are eager and willing to vote in ways we've deemed harmful to the hub. Imagine the founding fathers of the United States giving voting power to citizens of the UK because the US was a "fork" of the UK. 

There has been some talk of "alignment" in other threads in here. To ensure maximum alignment, I believe the decision to exclude those not in alignment is good for the cause. 

Instead of worrying about appeasing the current sloshy-ness (which can be traced all the way to the titanic, fed reserve, etc. aforementioned) around the Cosmos Hub, a full fledged exclusion for some parties is appropriate. Those that discount the minority opinion as "conspiracy theory" will only be a burden on what is trying to be accomplished. 

In addition, we should be looking outward at how to bring those outside of crypto, who may share similar beliefs, into the sphere of an $ATOM1 constellation. Those who understand the scam of central banks and the like, are less likely to find ATOM1 if they have to sift through the grifter garbage patch elements that are the current Cosmos Hub. 

How do we bring new participants to crypto or cosmos > appeasing everyone around the hub

**Comment by discoverdefiteam** [comment]
> Otherwise, how can we possibly hope to achieve a constitutional majority? I mean, we probably will at some time, but it would be better if we could guarantee it somehow... that eventually a constitutional majority can be achieved. Not sure how this can be done without lowering the constitutional majority which I don't recommend.


[prop #82](https://www.mintscan.io/cosmos/proposals/82) & [prop #848](https://www.mintscan.io/cosmos/proposals/848), both provide an opportunity to verify the alignment of intentions for Gaia compared to the initial vision of the minimal hub Gaia set out to be. 
 
## Data for Prop 82:
- `71.02M` voted YES
- `58.71M` voted NO + NWV
- For every `1.00` NO + NWV, there was `1.2096746721` YES votes. 
## Data for Prop 848:
- `73.165M` voted YES.
- `68.336M` voted NO + NWV
- For every `1.00` NO + NWV, there was `1.070665535` YES votes. 


## Gauge Metrics
1. If you voted `YES` on `#848`, and `YES` on `#82`  , you get `additional 50% less` $ATOM1 as $ATOM
3. If you voted `YES` on `#848`, and `ABSTAIN` on `#82`, you get `21% less` $ATOM1 
4. If you voted `ABSTAIN` on `#848`, and `YES` on `#82`, you get `21% less`  $ATOM1. 
5. If you voted `NO + NWV` on`#848`,  and `NO + NWV` on`#82`, you get `the 92% from results of the other gauges, distributed evenly` as $ATOM1 

The goal in my example here is to find gauge metrics that:
- further reward aligned participants
- incentivize conviction voting, opposed to zombie voting
- modular to any slashing decision gauge for prop #848 


Understanding the dataset if we combined the two props will reveal if this gauge will serve beneficial or harmful to the genesis distribution and alignment.

**Comment by boliwar** [comment]
There are people who couldn't vote for #82 because there was no wallet at the time. But they voted on item #848.
What to do with them?

**Comment by COverS8** [comment]
Atom was distributed based on early contributions, there never was an airdrop. For this reason, the only option for latecomers has been to buy Atom token with their own funds. This is why I proposed in addition to 1:1 airdrop with criterias that smaller Atom holders would receive a multiplier. The goal would be to distribute "skin in the game" more evenly, which would also incentivize more participation in the governance and voting. The airdrop could be partially locked at the beginning, although there are many opinions with this, some see it as forcing, but from the project perspective vesting is about longer-term commitment. 

But in a nutshell, I'm hoping that AtomOne contributors could find some more equal and decentralized solutions for the distribution than e.g. 26,000 Atom stakeholder would get 26,000 Atom1 and let's say 100 Atom stakeholder would get 100 Atom1. 

I checked and didn't find any of our explorers so if someone could scrape out the data of staked Atom account balances in some kind of order to see what is the average stakeholdings per accounts
https://github.com/QuokkaStake/cosmos-wallets-exporter

**Comment by giunatale** [comment]
> There are people who couldn't vote for #82 because there was no wallet at the time. But they voted on item #848. What to do with them?

We would have to find a methodology to combine the snapshots at the two different times that actually go beyond this issue. Stakers might have exited as well prior to 848 and for most of them their staked ATOMs have either increased or decreased between the two snapshots. So there's some work to do there. Maybe @michaelfrazzy you could help design this?

In general I think something like @discoverdefiteam is a good starting point. Plust I also would like to know what others think about @COverS8 idea because I personally am not against it. Maybe worth considering following what @jaekwon and initially @AloisLIEN-SetC were proposing, which would be a "bonus" pre-mint for more aligned voting behavior instead of a slash.

**Comment by jaekwon** [comment]
> The goal would be to distribute "skin in the game" more evenly

I can't agree with this because it just incentivizes people who split their wallets using programs. Everything has to be pro-rata linear, no curves.

**Comment by MichaelFrazzy** [comment]
@giunatale @jaekwon more than happy to take a stab at how we could linearly weight between multiple proposals. A couple ideas come to mind (super basic starting examples), in order from lowest to highest airdrop multiplier: 

- 2 Yes
- 1 Yes, 1 Abstain 
- 1 Yes, 1 No
- 1 Yes, 1 No with Veto
- 2 Abstain 
Etc in this way up until 2 No with Vetos as the highest multiplier. 

With 2 proposals it'd get a little messier due to all the combos between them needing defined, but we could just assign an airdrop multiplier to each combo. Similar to Gnoland but assigning a greater variety of multipliers to the greater variety of vote options. 

The other simple option is we set a multiplier to each option per vote, then those 2 multipliers sum for the final multiplier. 

Example: a No With Veto vote is worth 1.0 per proposal, so if someone voted No with Veto both times that'd be 1.0 + 1.0 = x2.0. So they'd get 2x their snapshotted ATOM value during the airdrop. Or yes would be 0 or maybe 0.1, so if someone voted yes twice they'd get 0 or maybe x0.2 their snapshotted ATOM. 

We could always create an actual equation/algorithm of some kind if a simpler idea like these 2 doesn't end up covering everything.

**Comment by tintin2828** [comment]
Those who voted no with veto or no in the last proposal which qualified for GNO land to be considered and those who did not vote this proposal and got overwritten by a validator yes vote would be considered as not all had visibility on Jae's messaging for this time.

**Comment by giunatale** [comment]
@tintin2828 as I've replied in the issue you've opened

>  due to how governance work it's a duty of stakers to override their validator's vote if it does not align, or re-delegate accordingly. You inherit your validator vote otherwise. This is the social contract of current cosmos governance. We can't alter this assumption when coming up with the distribution.

and regarding proposal 69 and gno.land, you can see from the discussion above we are agreeing we should consider only prop 82 and 848.

@MichaelFrazzy I kind of like the idea from @discoverdefiteam where you redistribute the amount you slash from yes-voters to others, and this way you can pretty much do this mirroring more or less the current ATOM supply.

> The other more straight forward option is we set a multiplier to each option per vote, then those 2 multipliers sum for the final multiplier.

I would say let's do this. Seems also easier to do in the context of having to cross-correlate two snapshots where the account-set will not overlap and staked amounts will also be different.

**Comment by MichaelFrazzy** [comment]
@giunatale I really like the idea to add slashed tokens back into the airdrop, sounds great. That combined with the multiplier sum between two (or however many) different proposals. 

Slashing made a little less of an impact than expected when we simulated prop 69. Adding the slashed amount to strictly No with Veto (maybe No voters also) would magnify that impact and overall alignment. As soon as we get the validator JSON I can create an initial sim of this asap!

**Comment by jaekwon** [comment]
> So I suggest for sub 5,000 Atom stakeholders some kind of multilpliers that the distribution would be more equal

This is game-able, so we cannot make such a modification, no project should. It just rewards people who splits their wallets in anticipation. If you want a project that incentivizes more equality it should be a project that splits off from AtomOne, or a project like the Decentralists which is meant to be different than Gaia/AtomOne's PoS model. The purpose of AtomOne is to double down on the original design goals and to complete it.

**Comment by jaekwon** [comment]
> @MichaelFrazzy I kind of like the idea from @discoverdefiteam where you redistribute the amount you slash from yes-voters to others, and this way you can pretty much do this mirroring more or less the current ATOM supply.

This negates the goal of turning a punishment into a reward. It is more apparently a punishment for those who voted YES; a psychological effect. Also while this will make sense initially it will become a cause of confusion later as the supply changes between the two.

**Comment by jaekwon** [comment]
> Don't you think you're overcomplicating things with these ratios and percentages and stuff?
> People made a choice by voting.
> Those who didn’t vote are not interested.
> Who voted “YES” and take airdrop ATOM1 this let the goat into the garden. They (YESsers) love commercial things and will buy ATOM1.



> @giunatale @jaekwon more than happy to take a stab at how we could linearly weight between multiple proposals. A couple ideas come to mind (super basic starting examples), in order from lowest to highest airdrop multiplier:
> 
>     * 2 Yes
> 
>     * 1 Yes, 1 Abstain
> 
>     * 1 Yes, 1 No
> 
>     * 1 Yes, 1 No with Veto
> 
>     * 2 Abstain
> 
> 
> Etc in this way up until 2 No with Vetos as the highest multiplier. We could even make it a perfectly linear scale like Jae mentioned.
> 
> With 2 proposals it'd get a little messier due to all the combos between them needing defined, but we could just assign an airdrop multiplier to each combo. Similar to Gnoland but assigning a greater variety of multipliers to the greater variety of vote options.
> 
> The other more straight forward option is we set a multiplier to each option per vote, then those 2 multipliers sum for the final multiplier. Basically the same idea just viewed a little differently.
> 
> Example: a No With Veto vote is worth 1.0 per proposal, so if someone voted No with Veto both times that'd be 1.0 + 1.0 = 2.0. So they'd get 2x their snapshotted ATOM value during the airdrop. Or yes would be 0 or maybe 0.1, so if someone voted yes twice they'd get 0 or maybe x0.2 their snapshotted ATOM.
> 
> We could always create an actual equation/algorithm of some kind if a simpler idea like these 2 doesn't end up covering everything. But maybe we don't even need to get much more complex.

Just calculate how much each should get for 82 and 848 and blend the results by address. What you suggested doesn't work because the amount that one has before and after is different, and, people could have changed their addresses; right? 

The relative weighting between 82 and 848 could also be not 1:1. Maybe we just use the nominal ATOM amounts for each and say that 848 should be weighted more because it is more recent.

**Comment by MichaelFrazzy** [comment]
@jaekwon what do you mean by turning punishment into reward in this case? Are you proposing to not slash at all or to slash and burn opposed to slash and redistribute to no voters?

And sounds great, seems like we have the same conclusion. I definitely ranted out the thought process a bit more but essentially that's what @giunatale and I concluded. Having it so that each proposal is worth a different amount that simply get summed together. I'll definitely set up the sim so we can quickly mess around with the weights too, that weighting option has been the part I'm most excited about with this sim haha.

**Comment by jaekwon** [comment]
> @jaekwon what do you mean by turning punishment into reward in this case? Are you proposing to not slash at all or to slash and burn, opposed to slash and redistribute to no voters?

I'm proposing that we ONLY slash the ICF.
See comment above https://github.com/atomone-hub/genesis/issues/12#issuecomment-1831327478
Nobody else gets slashed.

**Comment by jaekwon** [comment]
> I kinda like the idea of a slash, it delivers a powerful message.

Does the ICF and all YES voters deserve the same penalty? 

I'd say the ICF deserves to get slashed because they aren't doing a satisfactory job, and it was a foundation made for the chain.

I think whether one voted YES or NO has more to do with, were they exposed to the counter-arguments against 848. And it would be better that they learn that they need to be more proactive in understanding proposals before voting on them, and then participate in AtomOne, than to turn them away and create tension, losing a supporter potentially forever.

It would be better if we included the YES voters (many who have expressed to me that they have changed their mind after learning about our point of view) to a lesser extent than the NO voters, and allow them to participate given that they agree to abide by the constitution after having understood the rationale and spirit of it. And make sure there are ways for voters to be better educated about each proposal, through ways to ensure that proposals AND the best arguments for and against are all broadly shared via on-chain mechanisms and specified in the constitution.

**Comment by jaekwon** [comment]
It's in the README.md. Instead we create a 10% premine. Please read the draft first. https://github.com/atomone-hub/genesis#genesis-distribution (though it is a draft and not complete nor consistent, it is the current working state.

**Comment by MichaelFrazzy** [comment]
> It's in the README.md. Instead we create a 10% premine. Please read the draft first. https://github.com/atomone-hub/genesis#genesis-distribution (though it is a draft and not complete nor consistent, it is the current working state.

Ah thank you! I was only reading the comments. I'll set up soon PRs as soon as everything is in a good place, happy to help with any docs too

**Comment by jaekwon** [comment]
> I have written down is a 0.2 multiplier if someone voted yes on both proposals.

We can't do this because it isn't fair for anyone who moved addresses. This isn't fair.
Therefore, let's just keep these independent and do a linear blend at the end.

**Comment by jaekwon** [comment]
> Instead of worrying about appeasing the current sloshy-ness (which can be traced all the way to the titanic, fed reserve, etc. aforementioned) around the Cosmos Hub, a full fledged exclusion for some parties is appropriate. Those that discount the minority opinion as "conspiracy theory" will only be a burden on what is trying to be accomplished.

I have to say I do love the principle. Let's encourage these people to sell what they get of their own accord. Then there will be less harsh feelings, and they will regret their own actions down the line ;)

> In addition, we should be looking outward at how to bring those outside of crypto, who may share similar beliefs, into the sphere of an $ATOM1 constellation. Those who understand the scam of central banks and the like, are less likely to find ATOM1 if they have to sift through the grifter garbage patch elements that are the current Cosmos Hub.

Fully agreed. Please make a new issue to start the discussion about how to attract the right people into our community, those who are aligned and want to participate as equals.

**Comment by COverS8** [comment]
> > So I suggest for sub 5,000 Atom stakeholders some kind of multilpliers that the distribution would be more equal
> 
> This is game-able, so we cannot make such a modification, no project should. It just rewards people who split their wallets in anticipation. This taints the distribution; a priori everyone knows that the distribution was distorted such that, of the whales, the ones that hide better are rewarded while the whales who are more transparent about their holdings are punished. Incentivizing loophole-perverts doesn't seem like a good idea. While we might get the effect you desire (plus the loophole-pervert whales avoiding penalties) it creates the precedent of making such a decision (that incentivizes lying, in essence).
> 
> If you want a project that incentivizes more equality it should be a project that splits off from AtomOne, or a project like the Decentralists which is meant to be different than Gaia/AtomOne's PoS model. It is more directly associated with the goal you want to accomplish.
> 
> If you really think the sub-xxxx reward is a good idea, I think it should be a new constitutional chain that pre-disclaims in the constitution that large accounts will be slightly punished to incentivize breaking all wallets down into smaller chunks. But if this was good, the chain mechanism itself should be punishing non-compliant large-wallets regularly, not once in a while upon airdrops.

Agree, would be gameable bc we don't have the tools to define multiple accounts from one main account if those are not created from one derivation path I guess. And thanks for giving your views as smarter one for the issues!

**Comment by MichaelFrazzy** [comment]
> > I have written down is a 0.2 multiplier if someone voted yes on both proposals.
> 
> 
> 
> We can't do this because it isn't fair for anyone who moved addresses. We shouldn't be punishing wallet migrations. Therefore, let's just keep these independent and do a linear blend at the end. 
> 
> 
> 
> It would be interesting to see whether the new distribution passes or fails any prior proposals with a constitutional majority. 

Separate values per airdrop so that even if they used 2 separate addresses, they'd get 0.2. Or whatever amount we assign per proposal, could be 0.05 for one and 0.15 for another (just throwing out random multipliers for illustration). That way even if they used two separate wallets they receive the same total as someone who used one

**Comment by jaekwon** [comment]
> > Fully agreed. Please make a new issue to start the discussion about how to attract the right people into our community, those who are aligned and want to participate as equals.
> 
> Whattabout Airdrop2, after Airdrop1 with own criterias to attract more ppl to Atom-One? But I think the AtomOne hub's main attraction will stems from the effort to address and improve the flaws that have disappointed Atom stakeholders in the Cosmos hub. Perhaps a survey could be conducted to list the concerns from the Cosmos hub community, and those deemed viable suggestions by AtomOne contributors would be fixed in the fork far as possible while more challenging and complicating ones would be processed after the launch. At the same keeping up the Cosmos hub community what will be fixed and improved which causes natural interest. And also listen to what they don't like, as long as it's in line with the AtomOne hub vision.

In a way Prop #82 distribution inclusion should be decided by NO or NO+weighted(nonvoter+ABSTAIN voting sets, and is itself the first decision to make. I think this will pass easily. This should not be seen so much as an additional airdrop, but a linear blending of proposal-based-distributions. But as for additional airdrops, I think it would be most appropriate for the chain to answer that after genesis, but that we add a way for this to happen (with consideration for consistency with $PHOTON and other concerns) through the constitution, and that the constitution should should specify the conditions in which this is allowed. It would make sense to allow, for a limited % of atom inflation, and there would need to be clear proof of value for the dilution.

- [ ] TODO: consider: a Constituional Majority should be required for such an inflation to pass, and the inflation shouldn't affect $PHOTON holders at all (this is an dilution only of non-$PHOTON $ATOM1 holders/stakers).

**Comment by floridamanfintech** [comment]
> Fully agreed. Please make a new issue to start the discussion about how to attract the right people into our community, those who are aligned and want to participate as equals.

@jaekwon 

I've been brainstorming and working on this the last few days. I should be ready to open the topic in a few days

**Comment by giunatale** [comment]
> 1. How to deal with weighted votes ?

What's the problem about going pro-rata as also suggested by @albttx here: https://github.com/atomone-hub/genesis/issues/18#issuecomment-1840901384?

You essentially separate the stake using the % of the weighted vote. 
If you voted 50% yes, 40% no and 10% NWV, the 50% of your stake is counted as yes, 40% as no and 10 as nwv.

> We could also do more complicated things like propagate the weight to the distribution algorithm, and apply the weight to the account balance bonus or slash. But when we combine that with the fact that we could use more than one proposal to define the distribution, I think it gets way too complicated.

I don't think it is. I think this is the actually easier way since coding this I don't see it as particularly hard. It all depends how you implement it.

**Comment by giunatale** [comment]
> Do they really deserve to get slashed down to 0? Why not 10%? 20%? 30%?

Not 0. I think something between 30-50% would be sufficient. The amount they get slashed, will (potentially, if we follow this approach) be redistributed to no votes so if the "original" voting distribution was 50:50 it would become more or less 25:75 if its 50% slash. We can then maybe count 80% unstaked ATOM and 100% with no bonus for abstainers.
But probably this should be actually data-driven, tweaked upon looking at how example distributions might affect reaching CM as stated at point 3-4 at the bottom of the comment.

> Do we really want to risk severing the relationship with EVERYONE who voted YES?

I don't think they care, most of them would probably just dump their $ATOM1. I am sorry, I will be a bit "harsh" here, so I'll remark before that this is *strictly* my personal opinion and it comes from a "hot head". While I think *some* yes voters might truly change their mind, I think we have to distinguish (but it's nuanced and probably impossible to truly do) who is right now interested in embarking in this experiment, and who is just after the prospect of an airdrop.

TLDR; I think ultimately, yes voters should not be slashed to 0 (but if they voted yes on both 82 and 848 they should probably be slashed significantly) but they should be slashed.

> What are the other mechanisms by which $ATOM holders can acquire $ATOM1 (or get voting power)

On the market? 🤣 jokes aside, I think I said it earlier, maybe $ATOM holders can be somehow incentivized to bring $ATOMs
over to be bonded for the $PHOTON version of $ATOM (whatever that is called, say phATOM for now). Not sure how (and if they actually get phATOM down the line), but all I am saying is that the $ATOM that might be used to acquire $ATOM1 should be accumulated on AtomOne for the "partyhub" idea. Not "burned" or anything like that. Probably that was obvious but nonetheless..

**Comment by jaekwon** [comment]
In the end what we need is a page where you can enter your address, and the page will consult the data and tell you that you get XYZ $ATONE because:

prop848( weighted(YES:50%,NO:50%,NWV:0% = 2.0average)x 1200ATOM = 2400ATONE) + prop82( NWV(2.1) x 100ATOM = 210ATONE ) = 2610ATONE

The real version should show quite a bit like the blend resulting from your delegations, as well as the list of delegations, and it would be nice to show how the blend ratios are derived from the table of delegations, all in one page.

We should do the same for gno.land.

**Comment by tintin2828** [comment]
Jae, That $ATONE calculation looks fine if agreed, but would have been nicer if NWV for 82 could have had a bit more significance 
but do not agree that we should do the same with gno.land ? unless I misunderstood or it will be the other way round for gno.land similar to example u have provided 
prop82( weighted(YES:50%,NO:50%,NWV:0% = 2.0average)x 1200ATOM = 2400ATONE) + prop848( NWV(2.1) x 100ATOM = 210ATONE ) = 2610ATONE
Think we already promised how gno.land will be distributed and the criteria was communicated. It might not be fair to change what was agreed...

**Comment by cosmosatomatom1** [comment]
Shouldn't we calculate the cases where the validator voted for prop848 and the cases where the person voted directly? If Cosmos voted directly from the perspective of realizing democracy, the weights of yes, no, nwv, and abstain would have to change.

**Comment by giunatale** [comment]
> Shouldn't we calculate the cases where the validator voted for prop848 and the cases where the person voted directly? If Cosmos voted directly from the perspective of realizing democracy, the weights of yes, no, nwv, and abstain would have to change.

https://github.com/atomone-hub/genesis/issues/12#issuecomment-1836984602

As I've written before

> due to how governance works it's a duty of stakers to override their validator's vote if it does not align, or re-delegate accordingly. You inherit your validator vote otherwise. This is the social contract of current cosmos governance. We can't alter this assumption when coming up with the distribution.

If we treat the two cases differently, we could end up benefitting lazy voters (that didn't vote yes themselves but inherited the yes from validators) and I don't think this is right.

Also, while it's true that governance needs improvements, striving for a "democratic" system within PoS is not really achievable without crippling the PoS system itself. PoS is *not* democracy
